File: MeetingLocation.Lua/Entities/CityEntity.lua
local module = {}

module.__index = module

function module.new(cityId, stateId, name)
	local entity = setmetatable({}, module)
	
  entity.CityId = cityId
  entity.StateId = stateId
  entity.Name = name
	
	return entity
end

function module:Print()
  print(self.CityId, self.StateId, self.Name)
end

return module

File: MeetingLocation.Python/Entities/CityEntity.py
class CityEntity:
    def __init__(self, city_id, state_id, name):
        self.CityId = city_id
        self.StateId = state_id
        self.Name = name

    CityId = 0
    StateId = 0
    Name = ""

    def __str__(self):
        return f" {self.CityId}, {self.StateId}, {self.Name}"
